### PR TITLE
TOMEE-2761 Add Portuguese translation for examples/javamail

### PR DESCRIPTION
**Pull request by nishantraut**

### CODE REVIEW

File: examples/javamail/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,201 @@
+:index-group: Unrevised
+:jbake-type: page
+:jbake-status: status=published
+= Javamail API
+
+Este é apenas um exemplo simples para demonstrar um uso muito básico do
```
Comment (marcoantoniobferreira) [inline examples/javamail/README_pt.adoc]: muito básico "da" API instead "do" API

File: examples/javamail/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+API. Deve ser o suficiente para você começar a usar o java mail
+pacotes.
+
+== Um serviço REST simples usando a API Javamail
+
+Aqui vemos um terminal RESTful muito simples que pode ser chamado com um
```
Comment (marcoantoniobferreira) [inline examples/javamail/README_pt.adoc]: "uma" mensagem instead "um" mensagem.

File: examples/javamail/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+mensagem a ser enviada por e-mail. Não seria difícil modificar o aplicativo
+para fornecer opções de configuração mais úteis. Como está, isso não enviará
+qualquer coisa, mas se você alterar os parâmetros para corresponder ao seu servidor de correio
+você verá a mensagem sendo enviada. Você pode encontrar muito mais detalhado
+informações sobre o
+https://java.net/projects/javamail/pages/Home#Samples[Javamail API here]
```
Comment (marcoantoniobferreira) [inline examples/javamail/README_pt.adoc]: Javamail API "aqui" instead "here"

File: examples/javamail/README_pt.adoc [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+    public void post() throws IOException {
+        final String message = WebClient.create("http://localhost:4204").path("/test/email/").post("Hello General", String.class);
+        assertEquals("Failed to send message: Unknown SMTP host: your.mailserver.host", message);
+    }
+}
+....
+
+#Corrida
+
+A execução do exemplo é bastante simples. No diretório "javamail-api" corre:
```
Comment (marcoantoniobferreira) [inline examples/javamail/README_pt.adoc]: "execute" instead "corre"

**Comment by nishantraut** [comment]
> Hi Nishant! Great translation with a few points of attention =)

File: examples/javamail/README_pt.adoc (hunk 2 above)
Comment (nishantraut) [reply examples/javamail/README_pt.adoc]: updated

File: examples/javamail/README_pt.adoc (hunk 3 above)
Comment (nishantraut) [reply examples/javamail/README_pt.adoc]: updated

File: examples/javamail/README_pt.adoc (hunk 4 above)
Comment (nishantraut) [reply examples/javamail/README_pt.adoc]: updated

**Comment by nishantraut** [comment]
> Hi Nishant! Great translation with a few points of attention =)

Thanks @marcoantoniobferreira for review, i have updated PR.

**Comment by monsonhaefel** [comment]
Give another quick look, @marcoantoniobferreira  and if its good let us know so it can be merged!

**Comment by marcoantoniobferreira** [comment]
> Amazing!

It can be merged @monsonhaefel !

File: examples/javamail/README_pt.adoc [hunk 5]
```diff
@@ -0,0 +1,201 @@
+:index-group: Unrevised
+:jbake-type: page
+:jbake-status: status=published
```
Comment (cesarhernandezgt) [inline examples/javamail/README_pt.adoc]: I would suggest chaning this line to:
`:jbake-status: published`

Comment (nishantraut) [reply examples/javamail/README_pt.adoc]: fixed

**Comment by nishantraut** [comment]
> Greate contribution @nishantraut , I just left a comment on the document headers.

Thanks @cesarhernandezgt i have fixed review comment

**Pull request merged**